### PR TITLE
Sonarqube refactoring

### PR DESCRIPTION
**Pull request by MaximilianV**
Some refactorings to satisfy SonarQube even more

### CODE REVIEW

File: EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java [hunk 1]
```diff
@@ -17,18 +17,22 @@
...
 import org.apache.wicket.markup.html.list.ListItem;
 import org.apache.wicket.markup.html.WebMarkupContainer;
-import org.apache.wicket.model.IModel;
-import org.apache.wicket.model.LoadableDetachableModel;
-import org.apache.wicket.model.Model;
-import org.apache.wicket.model.PropertyModel;
-import org.apache.wicket.model.StringResourceModel;
 import org.apache.wicket.validation.validator.RangeValidator;
 
 import java.util.HashMap;
```
Comment (hpibptlab) [inline EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java]: MINOR Remove this duplicated import. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by hpibptlab** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [GeneratePanel.java#L39](https://github.com/bptlab/Unicorn/blob/a72179401a0ac6081979bd19f5d83a359b09aa88/EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java#L39): Remove this duplicated import. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

File: EapImport/src/main/java/de/hpi/unicorn/importer/json/JsonExporter.java [hunk 2]
```diff
@@ -72,8 +82,8 @@ public File generateExportFileWithDependencies(final EapEventType eventType) {
             writer.flush();
             // close the stream
             writer.close();
-        } catch (final IOException e1) {
-            e1.printStackTrace();
+        } catch (final IOException e) {
+            logger.warn("Error while exporting JSON.", e);
```
Comment (frederike-ramin) [inline EapImport/src/main/java/de/hpi/unicorn/importer/json/JsonExporter.java]: There should also be something returned here, right?

**Comment by hpibptlab** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [GeneratePanel.java#L39](https://github.com/bptlab/Unicorn/blob/41bb7eb8de1720e558ae01222e852ce7f6b9856b/EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java#L39): Remove this duplicated import. [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Pull request merged**